From 185282ff9da4bc8eb79ff613faf4829479a5f574 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 3 Feb 2019 22:08:59 +0900
Subject: [PATCH 3/7] Convert to UTF-8

This is very odd to see this file in non UTF-8 format.
So converted to UTF-8.

Signed-off-by: Osamu Aoki <osamu@debian.org>
---
 depgraph/master.depword | 38 +++++++++++++++++++-------------------
 1 file changed, 19 insertions(+), 19 deletions(-)

diff --git a/depgraph/master.depword b/depgraph/master.depword
index d897485..bb760f4 100644
--- a/depgraph/master.depword
+++ b/depgraph/master.depword
@@ -1,37 +1,37 @@
 # UTF-8
 #
-# ��°�������������
+# 付属語の列を定義する
 #
-# indepword.txt�˳��ʻ줴�Ȥκǽ�ΥΡ��ɤ�������Ƥ��롣
+# indepword.txtに各品詞ごとの最初のノードが定義してある。
 #
-# ����ɽ
+# 活用表
 # \\include conjugate.table
-# ���Ѥ�����
+# 活用するもの
 # \\include conjugate.depword
-# ���Ѥ��ʤ����
+# 活用しないもの
 # \\include fix.depword
 #
-# ̾��
+# 名詞
 # \\include noun.depword
 # \\include noun-variant.depword
-# ����
+# 副詞
 # \\include av.depword
-# ư��
+# 動詞
 # \\include v.depword
-# ���ƻ�
+# 形容詞
 # \\include a.depword
-# ����ư��
+# 形容動詞
 # \\include ajv.depword
 #
 #
-@��³�� "" Sy@ Sy@�������
-# ���Ѵ��Τ�Ȥˤ����ʤ�ʤ��ΤǴ������ϻȤ�ʤ�
-#@������� "��" "��" "��" @
-@������� "" @
-@Ϣ�λ� "" St@
-#@������ "��" "����" "������" "��" "��" "��" "�Ȥ�" "�ä�" "��" "��" "��" "��" @ @���ѥ졼��
-@������� "" @
-@�Ĥ���� "" @
-@ñ���� "" @
+@接続詞 "" Sy@ Sy@間投助詞
+# 誤変換のもとにしかならないので間投助詞は使わない
+#@間投助詞 "ね" "な" "さ" @
+@間投助詞 "" @
+@連体詞 "" St@
+#@終助詞 "か" "かい" "かしら" "な" "ぞ" "ぜ" "とも" "って" "の" "わ" "や" "よ" @ @セパレータ
+@開き括弧 "" @
+@閉じ括弧 "" @
+@単漢字 "" @
 #
 #
-- 
2.20.1

